### DISCUSSION TITLE
general help / list users route

### DISCUSSION THREAD

**Discussion by luismanson** [fastapi-users/fastapi-users#451] [Q&A] [answered]
Hello, I have a few questions I cant solve for myself and need help with,
* I want to implement a simple user manager in my app but saw there are no routes to get a List of them. What do i need to implement in order to get a route which returns a List of Users?
 
* I noticed that the working example for SqlAlchemy is a bit different than what is show in FastApi tutorial, can someone point me in what do i need to learn to understand their differences and make my own choice?

Thanks

**Comment by frankie567** [comment, marked as answer]
Hi @luismanson!

Sorry for the respnse delay!

* The list users route depend on you, and how you want your API to work (pagination, filtering handling...). There used to be this route implemented in FastAPI Users but we removed it because it was too opinionated for the common case. Basically, you'll make a request to your database, instantiating the results into Pydantic models and return them.
* There is no big difference between what is done in the FastAPI tutorial and what we do here. The configuration of FastAPI Users is actually convenient logic that does basically the same thing as you would do normally. However, for your own route and models, you should follow the FastAPI tutorial, nothing special there 👍 FastAPI Users is just a shortcut, but it remains 100% FastAPI.

Cheers!
